plugins/inputs/suricata/README.md

```diff
@@ -0,0 +1,135 @@
+# Suricata plugin for Telegraf
+
+This plugin reports internal performance counters of the Suricata IDS/IPS
+engine, such as captured traffic volume, memory usage, uptime, flow counters,
+and much more. It provides a socket for the Suricata log output to write JSON
+stats output to, and processes the incoming data to fit Telegraf's format.
+
+### Configuration:
+
+```toml
+[[input.suricata]]
+  ## Data sink for Suricata stats log.
+  # This is expected to be a filename of a
+  # unix socket to be created for listening.
+  # Will be overwritten if a socket or file
```

Comment:
This was included because initially there was an explicit unlink for any file at the given location, a behaviour that has been changed in the meantime. Will adjust this comment for consistency.